packages/code-infra/src/utils/credentials.mjs

```diff
@@ -0,0 +1,138 @@
+/**
+ * Secure credential storage for CLI tools
+ *
+ * This file was generated by Claude Code to provide secure credential storage
+ * for the MUI code infrastructure tools. It uses OS keychain/credential manager
+ * to store all credentials in a single JSON object.
+ *
+ * @generated by Claude Code
+ */
+
+import { AsyncEntry } from '@napi-rs/keyring';
+
+const KEYRING_SERVICE = 'mui-code-infra';
+const REGISTRY_KEY = '__credential_registry__';
```

Comment:
I wouldn't store everything under a single key. Just a key for github auth object. and a key for another.

I'd keep this module to a get and set method.